test/system/030-run.bats

```diff
@@ -1159,6 +1159,9 @@ EOF
     if is_rootless; then
         run ulimit -c -H
         max=$output
+        if [[ "$max" = "0" ]]; then
+            skip "ulimit -c == 0, cannot run this test"
+        fi
```

Comment:
given the test runs with 1000 this should check for less than 1000, or I guess change 1000 to 1 in the test below.

Reply:
I was being lazy, because to check for `-lt 1000` I would first need to check for `!= "unlimited"`. My assumption is that anyone who mucks with ulimit will set it to `0`, not some weird small number. Not a perfect assumption, but I was hoping a good-enough one.

Reply:
I mean yeah then just change the podman run below to use 1 and not 1000 then the check for only 0 is fine.